engine/events/hidden_objects/diglett_sculpture.asm

```diff
@@ -0,0 +1,21 @@
+BookOrSculptureText::
```

Comment:
This file is named after the Diglett sculpture but it isn't only for the Diglett sculpture. I think naming it after the more generic Pokemon bookshelf would be better.